### PR TITLE
WIP: Add mechanism for changing copyright information

### PR DESCRIPTION
**Pull request by adrianchivu**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-SDK-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-SDK [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-SDK-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### CODE REVIEW

File: CDP4Dal/DAL/Dal.cs [hunk 1]
```diff
@@ -62,11 +62,11 @@ protected Dal()
             this.MetaDataProvider = StaticMetadataProvider.GetMetaDataProvider;
             this.SetCdpVersion();
         }
-        
+
         /// <summary>
         /// Gets the supported version of the data-model
         /// </summary>
-        public Version DalVersion { get; private set; }
+        public Version DalVersion { get; set; }
```
Comment (samatrhea) [inline CDP4Dal/DAL/Dal.cs]: is changing the accesor to public necessary? if you change it here, that does not mean the serializer version gets updated i think. I propose to revert, or to make sure that the serializer version is updated as well.. We might be introducing complexity and unexpected behavior?

File: CDP4JsonFileDal/JsonFileDalUtils.cs [hunk 2]
```diff
@@ -41,10 +41,15 @@ internal static class JsonFileDalUtils
...
+        /// <summary>
+        /// The default copyright text to be included in the exchange header file.
+        /// </summary>
+        public const string DefaultHeaderCopyright = "Copyright 2016 © RHEA.";
+
         /// <summary>
         /// The copyright text to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderCopyright = "Copyright 2016 © RHEA.";
+        public static string ExchangeHeaderCopyright { get; set; }
```
Comment (samatrhea) [inline CDP4JsonFileDal/JsonFileDalUtils.cs:52]: i would not make this static, and why don't we allow the API user to update the complete `ExchangeFileHeader` object?

**Review by samatrhea — CHANGES_REQUESTED** [review]

File: CDP4Dal/DAL/Dal.cs (hunk 1 above)
Comment (adrianchivu) [reply CDP4Dal/DAL/Dal.cs]: Changed to protected.

File: CDP4JsonFileDal/JsonFileDalUtils.cs (hunk 2 above)
Comment (adrianchivu) [reply CDP4JsonFileDal/JsonFileDalUtils.cs]: Ok.

**Review by samatrhea — APPROVED** [review]

**Pull request merged**